data/json/items/resources/appliance_parts.json

```
@@ -73,5 +73,35 @@
     "weight": "5 kg",
     "volume": "20 L",
     "longest_side": "69 cm"
+  },
+  {
+    "id": "fanblade_metal",
```

```suggestion
    "id": "fanblade_metal",
```

As commented, needs to be moved